config.yaml

```diff
@@ -38,15 +38,15 @@ options:
         --runtime-config=batch/v2alpha1=true --profiling=true
       Note: As of Kubernetes 1.10.x, many of Kubelet's args have been deprecated, and can
       be set with kubelet-extra-config instead.
-  docker-logins:
+  proxy-extra-args:
```

Comment:
proxy-extra-args was removed by https://github.com/charmed-kubernetes/charm-kubernetes-worker/pull/4 and should not be reintroduced here